examples/shapes_and_collections/hatch_style_reference.py

```diff
@@ -0,0 +1,40 @@
+"""
+=====================
+Hatch style reference
+=====================
+
+Hatching (pattern filled polygons) is supported currently in the PS,
+PDF, SVG and Agg backends only.
```

Comment:
Picking up on @jklymak's comment

> Which backends don't support hatching?

```suggestion
Hatching (pattern filled polygons) is currently supported in the backends
PS, PDF, SVG and *Agg. The backends OSX, WX and *Cairo ignore hatching.
```